launcher/package/rpm/MultiMC5.spec

```diff
@@ -1,14 +1,20 @@
 Name:           MultiMC5
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 Summary:        A local install wrapper for MultiMC
 
 License:        ASL 2.0
 URL:            https://multimc.org
-BuildArch:      x86_64
+ExclusiveArch:  %{ix86} x86_64
+
+BuildRequires:  desktop-file-utils
+BuildRequires:  libappstream-glib
+Requires:       zenity %{?suse_version:lib}qt5-qtbase wget xrandr
+Provides:       multimc = %{version}
+Provides:       MultiMC = %{version}
+Provides:       multimc5 = %{version}
```

Comment:
need to explictly define the version of what we provide to ensure anything or anyone depending on a specific version will get the right version.
It is recommended to also use %{?_isa} but since this package is practically arch independent that doesn't matter    
https://docs.fedoraproject.org/en-US/packaging-guidelines/#renaming-or-replacing-existing-packages